bookmarks.html

```diff
@@ -116,7 +116,7 @@ <h2>What html element uses an alt tag and why do we use it?</h2>
     </main>
     <footer>
       <nav>
-        <ul class="nav--list">
+        <ul class="nav__list">
```

Comment:
In your navigation list you nested your `li` in `a`, but `li` should be the children of the list. An anchor should be nested inside the list item. :) [See caninclude.](https://caninclude.glitch.me/caninclude?child=a&parent=ul)

Reply:
Thank you very much, I will reconsider my structuring. Thank you for the "caninclude" link, too! …